src/root.component.tsx

```diff
@@ -1,14 +1,46 @@
 import React from "react";
-import { Route } from "react-router";
-import { BrowserRouter } from "react-router-dom";
+import { BrowserRouter, Route } from "react-router-dom";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import PatientChartSummary from "./summary/patient-chart-summary.component";
+import PatientBanner from "./summary/banner/patient-banner.component";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { LevelTwoRoutes } from "./summary/level-two-routes";
 
 function Root(props) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+  const [showPatientSummary, setShowPatientSummary] = React.useState(true);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient => {
+      setCurrentPatient(patient);
+      createErrorHandler();
+    });
+
+    return () => subscription.unsubscribe();
+  });
   return (
-    <BrowserRouter>
-      <Route to="/patient/:patientUuid/chart" component={PatientChartSummary} />
-    </BrowserRouter>
+    <main className="omrs-main-content">
```

Comment:
We now have two `<main>` elements, with one (PatientChartSummary) being inside of this one. The purpose of the `<main>` element is to indicate the **only** main content that a screen reader should focus on.

Additionally, it's a bit weird to put navigation (PatientBanner) into the main element, even though it doesn't scroll with the rest of the content.

My vote is to not put the `<main>` here but to instead put it inside of PatientChartSummary and LevelTwoRoutes